pyqtorch/hamiltonians/evolution.py

```diff
@@ -550,8 +556,12 @@ def jacobian(
             values = embedding(values)
 
         hamiltonian: torch.Tensor = self.create_hamiltonian(values, embedding)  # type: ignore [call-arg]
+        # note: trick for using GPSR with a same param_name used in many operations
```

Comment:
Probably best to harmonise with the other comment. Sorry for not having picked that up earlier.